### PR TITLE
[ACM-13266] - Enable mounting of arbitrary Secrets by alertmanager

### PR DESCRIPTION
**Pull request by philipgough**
https://issues.redhat.com/browse/ACM-13266 - allows a user to mount Secrets in alertmanager from the `advancedConfig` option in MCO. 

Following the pattern of prometheus-operator, Secrets will be mounted as a volume named secret-<secret-name>. Secrets are mounted into /etc/alertmanager/secrets/<secret-name> in the 'alertmanager' container.

### CODE REVIEW

File: operators/multiclusterobservability/pkg/rendering/renderer_alertmanager.go [hunk 1]
```diff
@@ -40,88 +40,105 @@ func (r *MCORenderer) newAlertManagerRenderer() {
...
+	alertManagerContainer := &spec.Containers[0]
+	configReloaderContainer := &spec.Containers[1]
+	oauthProxyContainer := &spec.Containers[2]
+	kubeRbacProxyContainer := &spec.Containers[3]
 
+	alertManagerContainer.ImagePullPolicy = imagePullPolicy
 	if *dep.Spec.Replicas > 1 {
 		for i := int32(0); i < *dep.Spec.Replicas; i++ {
-			args = append(args, "--cluster.peer="+
+			alertManagerContainer.Args = append(alertManagerContainer.Args, "--cluster.peer="+
```
Comment (saswatamcode) [inline operators/multiclusterobservability/pkg/rendering/renderer_alertmanager.go:82]: Can just use fmt.Sprint here instead of adding strings

**Review by saswatamcode — COMMENTED** [review]
Thanks!

File: operators/multiclusterobservability/pkg/rendering/renderer_alertmanager.go (hunk 1 above)
Comment (philipgough) [reply operators/multiclusterobservability/pkg/rendering/renderer_alertmanager.go:82]: yep. I didn't add this btw, just moved things around. Needs a rebase now anyway so will take care of it

**Review by saswatamcode — APPROVED** [review]

**Pull request merged**